app/views/icons/show.haml

```diff
@@ -6,6 +6,11 @@
     &raquo;
     = link_to "#{@icon.user.username}'s Galleries", user_galleries_path(@icon.user)
   &raquo;
+  - if @icon.galleries.count == 1
```

Comment:
This looks very curiously out of place, whatever could this be doing in a canon PR :P

Reply:
I VERY BELATEDLY ADMIT NOTHING